app/config/permissions.neon

```diff
@@ -356,24 +356,6 @@ permissions:
       actions:
           - assignExercise
       conditions:
-          - exercise.isAuthor
-          - group.isSupervisor
-          - group.isNotArchived
-
-    - allow: true
-      resource: group
-      actions:
-          - assignExercise
-      conditions:
-          - exercise.isSubGroupSupervisor
-          - group.isNotArchived
-
-    - allow: true
-      resource: group
-      actions:
-          - assignExercise
-      conditions:
-          - exercise.isPublic
```

Comment:
Seems like this condition was lost... is that correct?

Reply:
It is. New `exercise.assign` permission embeds this condition in `isSubGroupNonStudentMember` and `isSuperGroupAdmin`, although it is not visible in the permissions file.